web/viewer.html

```diff
@@ -56,6 +56,20 @@
     <script src="../src/display/font_loader.js"></script>
     <script src="../src/display/annotation_helper.js"></script>
     <script>PDFJS.workerSrc = '../src/worker_loader.js';</script>
+<!-- example showing the workerSrc blob delivery callback mechanism:
```

Comment:
Besides the comment above; adding "example" code to `viewer.html` does not seem like a good idea, since this file is used in production code.
Examples should be placed in this folder: https://github.com/mozilla/pdf.js/tree/master/examples.